CMake dependency is higher than 3.12 as listed in the README

I tried compiling on CentOS running and preparing with CMake 3.12, which didn't work, but then when I ran the exact same command with CMake 3.20, it compiled fine. 

If I have time later I'll try to pin down which version of CMake is actually required.

That seems consistent with what I would expect, since I read somewhere that CMake 3.13 added a new place it could look for targets.

When I compile on CMake 3.13.0

```
[root@server vvenc]# /root/cmakevs/cmake-3.13.0/bin/cmake --version
cmake version 3.13.0

CMake suite maintained and supported by Kitware (kitware.com/cmake).
[root@server vvenc]# /root/cmakevs/cmake-3.13.0/bin/cmake .
-- The C compiler identification is GNU 7.3.1
-- The CXX compiler identification is GNU 7.3.1
-- Check for working C compiler: /opt/rh/devtoolset-7/root/usr/bin/cc
-- Check for working C compiler: /opt/rh/devtoolset-7/root/usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /opt/rh/devtoolset-7/root/usr/bin/c++
-- Check for working CXX compiler: /opt/rh/devtoolset-7/root/usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- CMAKE_MODULE_PATH: updating module path to: /root/vvc/vvenc/cmake/modules
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Looking for pthread_create
-- Looking for pthread_create - not found
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE  
-- Configuring done
-- Generating done
-- Build files have been written to: /root/vvc/vvenc
```

When I try to compile with CMake 3.12.4

```
[root@server vvenc]# /root/cmakevs/cmake-3.12.4/bin/cmake --version
cmake version 3.12.4

CMake suite maintained and supported by Kitware (kitware.com/cmake).
[root@server vvenc]# /root/cmakevs/cmake-3.12.4/bin/cmake .
-- The C compiler identification is GNU 7.3.1
-- The CXX compiler identification is GNU 7.3.1
-- Check for working C compiler: /opt/rh/devtoolset-7/root/usr/bin/cc
-- Check for working C compiler: /opt/rh/devtoolset-7/root/usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /opt/rh/devtoolset-7/root/usr/bin/c++
-- Check for working CXX compiler: /opt/rh/devtoolset-7/root/usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- CMAKE_MODULE_PATH: updating module path to: /root/vvc/vvenc/cmake/modules
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Looking for pthread_create
-- Looking for pthread_create - not found
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE  
CMake Error at cmake/modules/vvencInstall.cmake:11 (install):
  install TARGETS given target "vvenc" which does not exist in this
  directory.
Call Stack (most recent call first):
  cmake/modules/vvencInstall.cmake:53 (install_targets)
  CMakeLists.txt:348 (include)


CMake Error at cmake/modules/vvencInstall.cmake:11 (install):
  install TARGETS given target "vvenc" which does not exist in this
  directory.
Call Stack (most recent call first):
  cmake/modules/vvencInstall.cmake:54 (install_targets)
  CMakeLists.txt:348 (include)


CMake Error at cmake/modules/vvencInstall.cmake:11 (install):
  install TARGETS given target "vvenc" which does not exist in this
  directory.
Call Stack (most recent call first):
  cmake/modules/vvencInstall.cmake:55 (install_targets)
  CMakeLists.txt:348 (include)


-- Configuring incomplete, errors occurred!
See also "/root/vvc/vvenc/CMakeFiles/CMakeOutput.log".
See also "/root/vvc/vvenc/CMakeFiles/CMakeError.log".
```

It seems to me that required CMake version is minimum 3.13.0.

--- a/README.md
+++ b/README.md
@@ -16,14 +16,14 @@ A working CMake installation is required for building the software.
 Download CMake from http://www.cmake.org/ and install it. The following targets are supported: Windows (Visual Studio), Linux (gcc) and MacOS (clang).
 
 ## How to build for Windows?
-In order to compile the software for Windows, Visual Studio 15 2017 or higher and cmake version 3.12 or higher are required. Install gnuwin32 that provides make for Windows. To build the software open a command prompt window, change into the project directory and use:
+In order to compile the software for Windows, Visual Studio 15 2017 or higher and cmake version 3.13 or higher are required. Install gnuwin32 that provides make for Windows. To build the software open a command prompt window, change into the project directory and use:
 
     make install-release
 
 This will create the statically linked release version of the encoder applications in the install/bin/release-static/ subdirectory.
 
 ## How to build for Linux/MacOS?
-In order to compile the software for Linux, gcc version 7.0 or higher and cmake version 3.12 or higher are required. For MacOS, Xcode and cmake version 3.12 or higher are required. To simplify the build process a Makefile with predefined targets is available. To build the VVenC encoder applications open a terminal, change into the project directory and use:
+In order to compile the software for Linux, gcc version 7.0 or higher and cmake version 3.13 or higher are required. For MacOS, Xcode and cmake version 3.13 or higher are required. To simplify the build process a Makefile with predefined targets is available. To build the VVenC encoder applications open a terminal, change into the project directory and use:
 
     make install-release
 

